pkg/testers/clusterloader2/cl2.go

```diff
@@ -0,0 +1,114 @@
+package clusterloader2
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/octago/sflags/gen/gpflag"
+	"k8s.io/klog"
+
+	"sigs.k8s.io/kubetest2/pkg/exec"
+	suite "sigs.k8s.io/kubetest2/pkg/testers/clusterloader2/suite"
+)
+
+type Tester struct {
+	Suites        string `desc:"Comma separated list of standard scale testing suites e.g. load, density"`
+	TestOverrides string `desc:"Comma separated list of paths to the config override files. The latter overrides take precedence over changes in former files."`
+	TestConfigs   string `desc:"Comma separated list of paths to test config files."`
+	Provider      string `desc:"The type of cluster provider used (e.g gke, gce, skeleton)"`
+	KubeConfig    string `desc:"Path to kubeconfig"`
```

Comment:
shouldn't we inject the kubeconfig env to all testers?

Reply:
(this also allows us to do multi-file for multi-cluster)

Reply:
we already do, this is an override. And yeah clusterloader2 doesn't support multi kubeconfig yet :/

Reply:
the usage text should probably clarify that this is an override?

Reply:
done